### ISSUE TITLE
nodetool status return a negative load

### ISSUE THREAD

**Issue by tzach** [scylladb/scylla#719]
```
Datacenter: datacenter1
=======================
Status=Up/Down
|/ State=Normal/Leaving/Joining/Moving
--  Address    Load       Tokens  Owns    Host ID                               Rack
UN  127.0.0.1  -1990250000 bytes  256     ?       37b3d89b-fca2-42b7-a15c-d3237a3f2bcc  rack1

Note: Non-system keyspaces don't have the same replication settings, effective ownership information is meaningless
```

Same issue as #688 and #687 ?

**Comment by amnonh** [comment]
What build are you using and how do you reproduce the problem?

**Comment by tzach** [comment]
@amnonh 

> What build are you using and how do you reproduce the problem?

scylla-server-0.13.2-20151216.43d3c5f.fc22.x86_64

**Comment by dorlaor** [comment]
@amnonh was it fixed later, @tzach can you test in 0.14?

**Comment by tzach** [comment]
Tested a few times with 0.17 and fail to reproduce
